proxy/http2/Http2ClientSession.h

```diff
@@ -292,6 +292,19 @@ class Http2ClientSession : public ProxyClientSession
     return half_close_local;
   }
 
+  bool
+  is_url_pushed(const char *url)
+  {
+    InkHashTableValue hash_value;
+    return ink_hash_table_lookup(h2_pushed_urls, url, &hash_value);
+  }
+
+  void
+  add_url_to_pushed_table(const char *url)
+  {
+    ink_hash_table_insert(h2_pushed_urls, url, NULL);
```

Comment:
Please use `nullptr` instead of `NULL`:)
( [C++ 11 Guidelines](https://cwiki.apache.org/confluence/pages/viewpage.action?pageId=65146830) )